with-crm-components/src/app/extensions/AssociationTableExtension.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { hubspot, Text } from '@hubspot/ui-extensions';
+import { CrmAssociationTable } from '@hubspot/ui-extensions/crm';
+
+// Define the extension to be run within the Hubspot CRM
+hubspot.extend(({ context }) => <Extension context={context} />);
+
+const Extension = ({ context }) => {
+  return (
+    <>
+      <Text format={{ fontWeight: 'bold' }}>Important Deals</Text>
```

Comment:
Suggestion: Add the filter criteria to help illustrate what the table is displaying. 
Something like "Open deals greater than $1000"